CHANGELOG.md.license

```diff
@@ -0,0 +1,2 @@
+SPDX-FileCopyrightText: 2024 SECO Mind Srl
+SPDX-License-Identifier: Apache-2.0
```

Comment:
Our markdown files are usually licensed under some creative commons instead of Apache